ifsbench/tests/test_util.py

```diff
@@ -46,3 +45,9 @@ def test_execute():
         text = 'abc\n' * 100
         execute(['echo', text], logfile=logfile)
         assert logfile.read_text() == text + '\n'
+
+        # Write to stderr
+        result = execute([sys.executable, '-c', 'import sys; print(\'foo bar\', file=sys.stderr)'], logfile=logfile)
+        assert logfile.read_text() == 'foo bar\n'
+        assert result.stdout == ''
```

Comment:
is there a way to mock time() in pytest to explicitly set the values it returns for repeated calls? Then you could also test the time.